### DISCUSSION TITLE
Discord?

### DISCUSSION THREAD

**Discussion by reverofevil** [peggyjs/peggy#293] [Ideas]
There's a lot of things to discuss, but Github Discussions is not really convenient for them.

Would you mind creating a Discord server?

**Comment by hildjj** [comment]
Would you be willing to do Slack?  I have to be in a bunch of Slack's anyway.  My second choice is probably Matrix.  I'll do Discord if there are any key folks who can't do anything else.

**Reply by hildjj** [reply]
(Note, to be clear, having a way to chat with higher bandwidth is something I support *strongly*)

**Reply by reverofevil** [reply]
From what I remember about using Slack for free a long time ago, it allows you to view only the last 10k messages. I've been in a situation when the message I needed disappeared just in a few weeks. I don't feel very positive about using (free) Slack ever since.

There are caveats with Discord too: its Electron app is sending way more telemetry than it should. But I consider using web version of Discord a minor nuisance compared to Slack's lack of stable conversation history.

**Reply by reverofevil** [reply]
Otherwise, I'll be happy to use whatever you guys end up choosing. I remembered Discord because it's what is usually chosen nowadays for communities around JS ([here](https://discord.gg/Gw83KVvW) or [here](https://discord.gg/typescript)).

**Comment by hildjj** [comment]
@Mingun do you have set of preferences?

**Reply by Mingun** [reply]
No, I do not use any messenger. Feel free to choose what you want, I hardly ever will join in any of them

**Comment by hildjj** [comment]
I realized I had a discord account, so I created an instance here: https://discord.gg/BMCfe2dF

**Comment by hildjj** [comment]
See #297.

**Comment by hildjj** [comment]
I also set up a notifications channel and connected github to it... I think.

**Reply by hildjj** [reply]
That worked, but now I've switched it to the whole org.  Does it still work?

**Comment by jcubic** [comment]
The invitation doesn't work.

**Comment by hildjj** [comment]
There's a widget on https://peggyjs.org/ that should work.  Invite links turn out only to be good for a week.
